neo-gui/neo-gui.csproj

```diff
@@ -736,7 +745,7 @@
     </COMReference>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Neo" Version="2.10.1" />
+    <PackageReference Include="Neo" Version="2.10.2.1" />
```

Comment:
Why `2.10.2.1`?

Reply:
Because it is the latest stable version from NuGet.
And in test, it works fine.

Is there any possible problem?